src/test/utils/utils.ts

```diff
@@ -18,7 +17,7 @@ export const verifyDownloadedChat = (
 ): void => {
   // get file from download folder
   const downloadsFolder = Cypress.config('downloadsFolder');
-  const filename = path.join(downloadsFolder, name);
+  const filename = [downloadsFolder, name].join('/');
```

Comment:
`path` doesn't work in the browser? or something like that? Maybe you can include it in the cypress scope it could work better?

Reply:
@pyphilia Yes it is related to using CRAv5 which uses webpackv5 that does not provide polyfills by default... I figured that since it is only to join a folder's name in a utility function of the tests it was not worth it to dig into how to add it.